corehq/apps/reports/standard/forms/reports.py

```diff
@@ -109,16 +90,26 @@ def _fmt_url(doc_id):
                     }
                 except NoReverseMatch:
                     return 'unable to view form'
-            
+
             def _fmt_date(somedate):
                 time = ServerTime(somedate).user_time(self.timezone).done()
                 return time.strftime(SERVER_DATETIME_FORMAT)
-            
-            return [_fmt_url(error_doc.get_id),
-                    error_doc.metadata.username if error_doc.metadata else EMPTY_USER,
-                    _fmt_date(error_doc.received_on),
-                    xmlns_to_name(self.domain, error_doc.xmlns, app_id=getattr(error_doc, 'app_id', None)) if error_doc.metadata else EMPTY_FORM,
-                    SubmissionErrorType.display_name_by_doc_type(error_doc.doc_type),
-                    getattr(error_doc, 'problem', None) or EMPTY_ERROR]
-
-        return [_to_row(error_doc) for error_doc in items]
+
+            if xform_dict['form'].get('meta'):
+                form_name = xmlns_to_name(
+                    self.domain,
+                    xform_dict.get('xmlns'),
+                    app_id=xform_dict.get('app_id'),
+                )
+            else:
+                form_name = EMPTY_FORM
+            return [
+                _fmt_url(xform_dict['_id']),
+                xform_dict['form']['meta']['username'] if xform_dict['form'].get('meta') else EMPTY_USER,
```

Comment:
seems like very similar logic to `form_name`, could use the same if else clause and make this list construction even tidier